### PR TITLE
Formatting and clarifications

### PR DESCRIPTION
**Pull request by apinnick**
No Jira or BZ ID.

Formatting fixes and clarifications from xref:https://github.com/openshift/openshift-docs/pull/33673

4.5+

Previews: https://deploy-preview-33872--osdocs.netlify.app/openshift-enterprise/latest/migration-toolkit-for-containers/advanced-migration-options-mtc.html#migration-mtc-cr-manifests_advanced-migration-options-mtc

https://deploy-preview-33872--osdocs.netlify.app/openshift-enterprise/latest/migration-toolkit-for-containers/advanced-migration-options-mtc.html#migration-about-migrating-applications-api_advanced-migration-options-mtc (only this section requires QE)

QE approved

### CODE REVIEW

File: modules/migration-excluding-resources.adoc [hunk 1]
```diff
@@ -26,7 +26,7 @@ $ oc edit migrationcontroller <migration_controller> -n openshift-migration
 apiVersion: migration.openshift.io/v1alpha1
 kind: MigrationController
 metadata:
-  name: migration-controller
+  name: <migration-controller>
```
Comment (XinRedhat) [inline modules/migration-excluding-resources.adoc:29]: @apinnick by default its name is `migration-controller`, nobody should modify it

**Comment by XinRedhat** [comment]
other looks good to me

File: modules/migration-mtc-cr-manifests.adoc [hunk 2]
```diff
@@ -47,16 +47,16 @@ kind: DirectImageStreamMigration
 metadata:
   labels:
     controller-tools.k8s.io: "1.0"
-  name: directimagestreammigration_name
+  name: <directimagestreammigration>
```
Comment (mikemckiernan) [inline modules/migration-mtc-cr-manifests.adoc:50]: Consider putting an underscore between the words, like you have for image_stream below.

**Review by mikemckiernan — APPROVED** [review]
Looks good.

**Pull request merged**